tap_zoom/client.py

```diff
@@ -84,7 +101,7 @@ def request(self,
         if url is None and \
```

Comment:
What is purpose of this `url is None` check ?

Reply:
If we don't add this check, it calls self.refresh_access_token function in loop and throws the error : 
```
RecursionError: maximum recursion depth exceeded while calling a Python object
```